enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -110,26 +106,58 @@ A `ServiceMonitor` object must be created in the namespace mentioned above and p
 
 ##### RBAC Requirements
 
-A role/rolebinding must be created that provides the Prometheus Operator ServiceAccount with the appropriate RBAC privileges to discover the `ServiceMonitor` and `PrometheusRule` resources in the newly created namespace.
+A role and rolebinding must be created that provides the OpenShift Monitoring `prometheus-k8s` ServiceAccount with the appropriate RBAC privileges to discover the `Services` in your operator's namespace. An example of the required RBAC can be seen below:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: 'prometheus-k8s',
+  namespace: 'openshift-monitoring',
+```
```

Comment:
What does ownership look like if multiple operators provide identical `RoleBindings` in a namespace? i.e. which CSV becomes the owner, all of them, none of them?